api-docs/release-notes/releases/cdbv3.10.0.rst

```diff
@@ -0,0 +1,43 @@
+.. version-3.10.0-release-notes:
+
+v3.10.0, October 27, 2020
+-------------------------
+
+What's new
+~~~~~~~~~~
+
+-  Upgrading a single instance will put both the instance and it's upgraded
+   replica into a VERIFY_UPGRADE state.
+
+   **Note:** Both the instance and upgraded replica will remain in this state
+   until the upgraded replica is detached from the instance.
+
+   This is done to keep data current in the upgraded replica while the
+   customer validates that their application doesn't have any issues with the
+   upgraded replica and plan a maintenance to disconnect their application from
+   the instance and connect to the upgraded replica.
+
+-  Scheduled backups can be setup to take a full backup every day instead of
+   taking a full one day a week and incrementals on the other days.
+
+-  On-demand upgrade of HA groups:
+
+   -  You can upgrade HA MySQL 5.1/5.6 to MySQL 5.7.
+   -  You can upgrade HA MariaDB 10/10.1 to MariaDB 10.4.
+   -  You can upgrade HA MariaDB 10.4 to MariaDB 10.4enc.
+
+-  General availability of backup copy functionality for on-demand and scheduled
+   backups.
+
+Resolved issues
+~~~~~~~~~~~~~~~
+
+-  Request for on-demand upgrade of single instance will be rejected if the
```

Comment:
Request for an on-demand upgrade of a single instance is rejected if the   [removed will and added articles]